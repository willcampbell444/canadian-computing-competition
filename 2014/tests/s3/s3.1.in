5
4
3
2
1
4
5
2
5
1
3
4
6
3
4
2
5
6
1
7
5
2
7
1
3
4
6
8
4
8
3
2
5
1
6
7
