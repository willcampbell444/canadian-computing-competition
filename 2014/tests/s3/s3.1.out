Y
N
Y
N
N
